src/execution_algorithms/kfac.cpp

```diff
@@ -87,9 +95,68 @@ KFAC::KFAC(
     m_disable_layers{std::move(disable_layers)},
     m_learning_rate_factor{learning_rate_factor},
     m_learning_rate_factor_gru{learning_rate_factor_gru},
-    m_compute_interval{compute_interval}
+    m_compute_interval{compute_interval},
+    m_distribute_precondition_compute{distribute_precondition_compute},
+    m_enable_copy_errors{enable_copy_errors},
+    m_enable_copy_activations{enable_copy_activations},
+    m_use_eigen_decomposition{use_eigen_decomposition},
+    m_use_KFAC_epoch{std::move(kfac_use_interval)}
 {}
 
+// KFAC::KFAC(KFAC const& other)
```

Comment:
If this class is not Copyable, that's fine, but can we just remove the dead code?